### PR TITLE
Fix - Responsive menu

### LINKED ISSUE #97: Updating the menu view + adding new menu point + updating pages behind menu points
- Adding new menu point `Weitere Anträge` and putting this on the second last position (see ranking of next point)

- updating the placeholder pages with the following links:
Über uns: https://www.deora.earth/volt-platform/about
Quadratical Voting: https://www.deora.earth/volt-platform/quadratic
Die Blockchain: https://www.deora.earth/volt-platform/blockchain
Sicherheit & Privacy: https://www.deora.earth/volt-platform/privacy
Weitere Anträge: https://www.deora.earth/volt-platform/extra-proposals
Wahlergebnisse: https://www.volt.deora.earth/results

- Menu view needs to be adjusted (made responsive).
Current behaviour, overlapping menu points.

image

### PR DESCRIPTION
**Pull request by MaxStalker**
Resolves #97

### CODE REVIEW

**Review by vrde — APPROVED** [review]
LGTM

File: src/volt/components/Menu/index.js [hunk 1]
```diff
@@ -8,11 +8,12 @@ const Menu = ({ onClose, account }) => (
...
-        <Item href="/page-2">Quadratical Voting</Item>
-        <Item href="/page-3">Die Blockchain</Item>
-        <Item href="/page-4">Sicherheit & Privacy</Item>
-        <Item href="/page-5">Wahlergebnisse</Item>
+        <Item target="_blank" href="https://www.deora.earth/volt-platform/about">Über uns</Item>
+        <Item target="_blank" href="https://www.deora.earth/volt-platform/quadratic">Quadratical Voting</Item>
+        <Item target="_blank" href="https://www.deora.earth/volt-platform/blockchain">Die Blockchain</Item>
+        <Item target="_blank" href="https://www.deora.earth/volt-platform/privacy">Sicherheit & Privacy</Item>
+        <Item target="_blank" href="https://www.deora.earth/volt-platform/extra-proposals">Weitere Anträge</Item>
+        <Item target="_blank" href="https://www.volt.deora.earth/results">Wahlergebnisse</Item>
```
Comment (vrde) [inline src/volt/components/Menu/index.js:16]: Without `www`

File: src/volt/components/Menu/index.js [hunk 2]
```diff
@@ -8,11 +8,12 @@ const Menu = ({ onClose, account }) => (
     <Container>
       <DeoraLogo />
       <MenuItems>
-        <Item href="/page-1">Über uns</Item>
-        <Item href="/page-2">Quadratical Voting</Item>
-        <Item href="/page-3">Die Blockchain</Item>
-        <Item href="/page-4">Sicherheit & Privacy</Item>
-        <Item href="/page-5">Wahlergebnisse</Item>
+        <Item target="_blank" href="https://www.deora.earth/volt-platform/about">Über uns</Item>
```
Comment (vrde) [inline src/volt/components/Menu/index.js:11]: All those links give 404, will they be available later?

**Review by vrde — CHANGES_REQUESTED** [review]

File: src/volt/components/Menu/index.js (hunk 1 above)
Comment (MaxStalker) [reply src/volt/components/Menu/index.js:16]: Website sports `www`. Do you think we shall still remove it?
https://www.deora.earth/#solutions

File: src/volt/components/Menu/index.js (hunk 2 above)
Comment (MaxStalker) [reply src/volt/components/Menu/index.js:11]: Last one will be available once the event is over

**Review by vrde — APPROVED** [review]

**Pull request merged**